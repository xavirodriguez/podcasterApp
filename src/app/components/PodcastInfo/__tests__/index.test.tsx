import { render } from '@testing-library/react';
import * as React from 'react';

import { PodcastInfo } from '..';

jest.mock('react-i18next', () => ({
  useTranslation: () => {
    return {
      t: str => str,
      i18n: {
        changeLanguage: () => new Promise(() => {}),
      },
    };
  },
}));

describe('<PodcastInfo  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(
      <PodcastInfo image={''} title={''} author={''} description={undefined} />,
    );

    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
