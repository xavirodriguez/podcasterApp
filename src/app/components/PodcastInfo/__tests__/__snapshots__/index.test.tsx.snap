// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PodcastInfo  /> should match snapshot 1`] = `
.c0 {
  width: 80%;
  margin: 0 auto;
  display: block;
}

<img
  class="c0"
  src=""
/>
`;
